appcues/src/main/java/com/appcues/ui/StateMachineDirectory.kt

```diff
@@ -33,9 +39,16 @@ internal class ModalStateMachineOwner(override val stateMachine: StateMachine) :
     override suspend fun reset() {
         stateMachine.stop(true)
     }
+
+    override suspend fun onConfigurationChanged() {
+        if (stateMachine.state is RenderingStepState) {
+            stateMachine.handleAction(Reset)
+        }
+    }
```

Comment:
following along - this is only handled in the ModalStateMachineOwner (modals and tooltips) as we reset the overlay content with the customer app changes config. With embeds, not needed, as those live within a customer view that has already been reset, presumably, and controls the layout of our composition within it.